internal/pkg/consensus/testing.go

```diff
@@ -239,14 +242,14 @@ func SeedFirstWinnerInNRounds(t *testing.T, n int, ki *crypto.KeyInfo, networkPo
 	}
 }
 
-func winsAtEpoch(t *testing.T, epoch uint64, ticket block.Ticket, ki *crypto.KeyInfo, networkPower, numSectors, sectorSize uint64, sectorInfos ffi.SortedPublicSectorInfo) bool {
-	signer := types.NewMockSigner([]crypto.KeyInfo{*ki})
-	wAddr, err := ki.Address()
-	require.NoError(t, err)
+func winsAtEpoch(t *testing.T, epoch abi.ChainEpoch, ticket block.Ticket, miner address.Address, networkPower, numSectors, sectorSize uint64, sectorInfos ffi.SortedPublicSectorInfo) bool {
 	em := ElectionMachine{}
 
 	// use ticket to seed postRandomness
-	postRandomness, err := em.GeneratePoStRandomness(ticket, wAddr, signer, epoch)
+	rnd := crypto.FakeRandomnessSource{Ticket: ticket}
+	entropy, err := mining.MakeElectionPoStChallengeSeedEntropy(epoch, miner)
+	require.NoError(t, err)
+	postRandomness, err := rnd.Randomness(context.Background(), acrypto.DomainSeparationTag_ElectionPoStChallengeSeed, epoch, entropy)
```

Comment:
Going by the interface this won't work as you need a signing key to generate the post vrf proof.

Continuing slack conversation here: PostRandomness is a bad name because it made you think this was the way to get it.  We should consider PRing a change to the spec to indicate that it is a VRF Proof derived off of the ticket chain with a miner sk, not something you can generate with publicly known values.

Reply:
postVRFProof is a nice boring and precise name

Reply:
postVRFProof is a nice boring and precise name